conf/application.conf

```diff
@@ -150,8 +150,8 @@ features {
   taskReopenAllowedInSeconds = 30
   allowDeleteDatasets = true
   # to enable jobs for local development, use "yarn enable-jobs" to also activate it in the database
-  jobsEnabled = false
-  voxelyticsEnabled = false
```

Comment:
eieie

Reply:
I will definitely remove this before merging, but will leave it in for now because the chances are high that I will forget to remove it before committing again

Reply:
since this convo was marked as resolved, I just wanted to leave a reminder here :)

Reply:
uff, thanks!